### PR TITLE
issue 5119 update readme

### LINKED ISSUE #5119: Update README
Grammar, stylistic and content improvements

### PR DESCRIPTION
**Pull request by vahram-papazyan**
This PR makes grammar, stylistic, and content improvements in the README

close #5119 

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Comment by vahram-papazyan** [comment]
@sosnovsky This pr is ready for review. 

The link to the Live tests isn't working (or redirects to a page I don't have access to). If there is a valid link, I can add it.

File: README.md [hunk 1]
```diff
@@ -1,55 +1,61 @@
...
+Get the [FlowCrypt](https://flowcrypt.com/) browser extension from the [FlowCrypt downloads](https://flowcrypt.com/download) page.
 
 ## Developers
 
 Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-browser.svg?key=d575b4ff-e35d-4217-9152-27cea9e72c19)
 
 Live tests: [![Live Tests](https://semaphoreci.com/api/v1/flowcrypt/flowcrypt-browser/branches/master/badge.svg)](https://semaphoreci.com/flowcrypt/flowcrypt-browser)
 
-This project is written in TypeScript. Browsers only understand JavaScript, so the project needs to be transpiled. You need to build the project the first time you download it, and build it after each change you make to see the result in the browser. To get started, please follow the instructions below:
+This project has been written in TypeScript. However, since browsers only understand JavaScript, the project needs to be transpiled to it. You need to build the project the first time you download/clone it, and then build it again after each change you make to see the result in the browser. To get started, please follow the instructions below:
```
Comment (sosnovsky) [inline README.md:13]: I think `is written` is more correct, as it means `project writing continues`, while `has been written` sounds like `project writing is finished`

File: README.md [hunk 2]
```diff
@@ -1,55 +1,61 @@
...
+4. Browse to the `flowcrypt-browser/build` folder and select the appropriate project version, either `build/chrome-consumer` or `chrome-enterprise`.
 
-Similarly, to load the extension in Firefox, follow these steps:
+Similarly, to load the extension in Firefox, please follow these steps:
 
 1. Open your Firefox browser and navigate to `about:debugging`.
 2. Click on the `This Firefox` tab.
 3. Click on the `Load Temporary Add-on` button located at the top-right corner.
-4. Browse to and select the appropriate folder, either `build/firefox-consumer` or `firefox-enterprise`, and select the `manifest.json` file.
+4. Browse to the `flowcrypt-browser/build` folder and select the appropriate project version, either `build/firefox-consumer` or `firefox-enterprise`, and select the `manifest.json` file.
```
Comment (sosnovsky) [inline README.md:46]: currently we build only `firefox-consumer` version, `firefox-enterprise` can be removed from this line

File: README.md [hunk 3]
```diff
@@ -1,55 +1,61 @@
...
 
-If you prefer, you can also use the `run_firefox` script (`npm run run_firefox`) included in the package.json file to run the Firefox extension in a separate instance without interfering with the production extension installed in your browser.
+If you wish, you can also use the `run_firefox` script (`npm run run_firefox`) included in the `package.json` file to run the Firefox extension in a separate instance without interfering with the production extension installed in your browser.
 
-Printing debug data to test logs can be done using special `Debug` class:
-https://github.com/FlowCrypt/flowcrypt-browser/tree/master/extension/js/common/platform/debug.ts#L7
+Printing debug data to test logs can be accomplished using a special `Debug` class: [github.com/FlowCrypt/flowcrypt-browser](https://github.com/FlowCrypt/flowcrypt-browser/tree/master/extension/js/common/platform/debug.ts#L7)
```
Comment (sosnovsky) [inline README.md:50]: let's make `special Debug class` a link, so it'll look like this:

Printing debug data to test logs can be accomplished using a [special Debug class](https://github.com/FlowCrypt/flowcrypt-browser/tree/master/extension/js/common/platform/debug.ts#L7)

File: README.md (hunk 1 above)
Comment (vahram-papazyan) [reply README.md:13]: What if we use "We develop the project..." in that case?

**Comment by sosnovsky** [comment]
> The link to the Live tests isn't working (or redirects to a page I don't have access to). If there is a valid link, I can add it.

It should be removed, badge for Mock tests shows result for all tests

File: README.md (hunk 1 above)
Comment (sosnovsky) [reply README.md:13]: Sounds good too 👍

Comment (vahram-papazyan) [reply README.md:13]: \+

**Comment by vahram-papazyan** [comment]
@sosnovsky This pr is ready for review

File: README.md [hunk 4]
```diff
@@ -1,55 +1,60 @@
...
+# FlowCrypt: Encrypt Gmail with OpenPGP
 
 ## Users
 
-Get [FlowCrypt](https://flowcrypt.com/) browser extension at: https://flowcrypt.com/download
+Get the [FlowCrypt](https://flowcrypt.com/) browser extension from the [FlowCrypt downloads](https://flowcrypt.com/download) page.
 
 ## Developers
 
 Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-browser.svg?key=d575b4ff-e35d-4217-9152-27cea9e72c19)
```
Comment (sosnovsky) [inline README.md:9]: let's remove `Mock tests: ` label (as badge displays result for all tests), and leave just image

File: README.md [hunk 5]
```diff
@@ -1,55 +1,60 @@
...
 
-Live tests: [![Live Tests](https://semaphoreci.com/api/v1/flowcrypt/flowcrypt-browser/branches/master/badge.svg)](https://semaphoreci.com/flowcrypt/flowcrypt-browser)
+We're developing the project in TypeScript. However, since browsers only understand JavaScript, the project needs to be transpiled to it. You need to build the project the first time you download/clone it, and then build it again after each change you make to see the result in the browser. To get started, please follow the instructions below:
```
Comment (sosnovsky) [inline README.md:11]: I think we agreed on `We develop` instead of `We're developing`, let's use it :)

File: README.md [hunk 6]
```diff
@@ -1,55 +1,60 @@
...
 
 To load the extension in Google Chrome, please follow these steps:
 
 1. Open your Chrome browser and navigate to `chrome://extensions/`.
-2. If not already enabled, toggle on the `Developer mode`    switch located at the top-right corner.
-3. Click on the `Load Unpacked` button.
-4. Browse to and select the appropriate folder, either `build/chrome-consumer` or `chrome-enterprise`.
+2. If it isn't already enabled, toggle on the **Developer mode**. The switch button is located at the top-right corner.
+3. Click on the `Load unpacked` button.
+4. Browse to the `flowcrypt-browser/build` folder and select the appropriate project version, either `build/chrome-consumer` or `chrome-enterprise`.
```
Comment (sosnovsky) [inline README.md:38]: `build/chrome-consumer` -> `chrome-consumer`
as user is already in `build` folder

File: README.md [hunk 7]
```diff
@@ -1,55 +1,60 @@
...
 
-Similarly, to load the extension in Firefox, follow these steps:
+Similarly, to load the extension in Firefox, please follow these steps:
 
 1. Open your Firefox browser and navigate to `about:debugging`.
 2. Click on the `This Firefox` tab.
 3. Click on the `Load Temporary Add-on` button located at the top-right corner.
-4. Browse to and select the appropriate folder, either `build/firefox-consumer` or `firefox-enterprise`, and select the `manifest.json` file.
+4. Browse to the `flowcrypt-browser/build` folder, open the `build/firefox-consumer` project version folder, and select the `manifest.json` file.
```
Comment (sosnovsky) [inline README.md:45]: `build/firefox-consumer` -> `firefox-consumer`
as user is already in `build` folder

File: README.md (hunk 4 above)
Comment (vahram-papazyan) [reply README.md:9]: \+

File: README.md (hunk 5 above)
Comment (vahram-papazyan) [reply README.md:11]: Done )

File: README.md (hunk 6 above)
Comment (vahram-papazyan) [reply README.md:38]: \+

File: README.md (hunk 7 above)
Comment (vahram-papazyan) [reply README.md:45]: \+

**Review by sosnovsky — APPROVED** [review]

**Pull request merged**